tests/python/unit_tests/test_optimizers/test_pybernyop.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2024 NWChemEx Community
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from structurefinder.pybernyopt import optimize_pyberny
+import chemist
+import unittest
+
+
+class Test_optimize_pyberny(unittest.TestCase):
+
+    def test_optimize_pyberny(self):
+        egy, new_geom = optimize_pyberny(self.mol)
```

Comment:
```suggestion
        mm = pp.ModuleManager()
        load_modules(mm)
        egy = mm.run_as("PyBerny",chemist.ChemicalSystem(self.mol))
```
Running the module is going to looks something like this now (I didn't test this so it may need some clean-up)